AnnoDesigner/App.xaml.cs

```diff
@@ -209,8 +181,21 @@ private async void Application_Startup(object sender, StartupEventArgs e)
                 _appSettings.Reload();
             }
 
-            //var updateWindow = new UpdateWindow();                
-            await _updateHelper.ReplaceUpdatedPresetsFilesAsync();
+            var anotherInstanceIsRunning = IsAnotherInstanceRunning();
+            if (anotherInstanceIsRunning && _appSettings.ShowMultipleInstanceWarning && await _updateHelper.AreUpdatedPresetsFilesPresentAsync())
+            {
+                //prevent app from closing, because there is no main window yet
+                Application.Current.ShutdownMode = ShutdownMode.OnExplicitShutdown;
+
+                //inform user that auto update is not applied
+                _messageBoxService.ShowMessage(Localization.Localization.Instance.GetLocalization("WarningMultipleInstancesAreRunning"));
```

Comment:
This would need much more changes to the update behavior.
But this check is only done when there are new update files present ([see check in line 185](https://github.com/AnnoDesigner/anno-designer/pull/405/files#diff-6a80cdbc60a5845df9a6d0ba0d7f759105952f466348372121edf9d7f20b84ebR185)).